From e55891ddea6548d31311c486f3f8ba37b1af84e5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Wed, 22 Feb 2023 15:00:46 +0100
Subject: [PATCH 1/2] python: use setuptools instead of distutils

distutils was deprecated in python3.10+ and is removed in python3.12.
Use setuptools instead.

Prompted by: https://bugzilla.redhat.com/show_bug.cgi?id=2154971
Documentation: https://peps.python.org/pep-0632/#migration-advice
---
 py_ext/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/py_ext/setup.py b/py_ext/setup.py
index c2160b6ce9..0bc35413b5 100644
--- a/py_ext/setup.py
+++ b/py_ext/setup.py
@@ -2,7 +2,7 @@ import os
 import re
 import sys
 from os.path import join, realpath
-from distutils.core import setup, Extension
+from setuptools import setup, Extension
 
 
 
-- 
2.39.1

